collectors/0/goproc.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import traceback
+
+import requests
```

Comment:
Will this require a dependency on [python-requests.deb/rpm](https://packages.debian.org/sid/python-requests)?